shell/ci/release/pre-release.sh

```diff
@@ -87,22 +81,38 @@ if [[ ! -e "$(get_repo_directory)/.goreleaser.yml" ]]; then
   exit 0
 fi
 
-app_version="v0.0.0-unstable+$(git rev-parse HEAD)"
-echo "Creating unstable release ($app_version)"
-
-make release APP_VERSION="$app_version"
-
 # If we're in dry-run mode, skip creating the release.
 if [[ $DRYRUN == "true" ]]; then
   exit 0
 fi
 
-# delete unstable release+tag if it exists
+# Check commit message on current branch
+# Prerelease to rc if the latest commit is chore: Release.
+COMMIT_MESSAGE=$(git log --format=%B -n 1)
+if [[ $COMMIT_MESSAGE == "chore: Release" ]]; then
+  # Pre-release to rc
+  echo "Creating prerelease to rc channel"
+  # Read the GH_TOKEN from the file
+  GH_TOKEN="$(cat "$HOME/.outreach/github.token")"
+  if [[ -z $GH_TOKEN ]]; then
+    echo "Failed to read Github personal access token" >&2
+  fi
```

Comment:
issue: according to this, `~/.outreach/github.token` is the legacy path:

https://github.com/getoutreach/devbase/blob/060fbed4a1457d1c6169f2ddfc90479bb56eaa58/shell/ci/auth/github.sh#L24-L26

You should be using `gh auth token`, I believe.

Reply:
Thanks, I updated to use `gh auth token`.